### PR TITLE
fix: DTA-79: update `disableDraftSpecs` property name

### PR DESCRIPTION
**Pull request by rosdb**

### CODE REVIEW

File: .changeset/many-schools-smile.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@detra-lab/esbuild-plugin-lit-css': patch
+---
+
+The property name disableDraftSpecs is misleading. It has been changed to includeDraftSpecs.
```
Comment (giotramu) [inline .changeset/many-schools-smile.md:5]:
```suggestion
The property name `disableDraftSpecs` is misleading. It has been changed to `includeDraftSpecs`.
```

File: src/index.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export interface LitCssPluginOptions {
    * You may disable them to speed up the process.
```
Comment (giotramu) [inline src/index.ts:16]: This message should be changed to:

```ts
/**
 * Lightning CSS compiles [specifications](https://lightningcss.dev/transpilation.html#draft-syntax) not yet 
 * available in browsers. You can include them and avoid transpilation to speed up the process.
 * Default: `false`.
 */
```

**Review by giotramu — CHANGES_REQUESTED** [review]

File: src/core/css-bundler.ts [hunk 3]
```diff
@@ -111,8 +111,8 @@ const browserlist = (
     )
   )
 
-const getDraftSpecs = (isDraftSyntaxDisabled: boolean): Drafts =>
-  isDraftSyntaxDisabled
+const getDraftSpecs = (isDraftSyntaxIncluded: boolean): Drafts =>
+  isDraftSyntaxIncluded
```
Comment (giotramu) [inline src/core/css-bundler.ts:115]: Here, we can use a similar namespace:

```suggestion
  isDraftSpecsIncluded
```

Thoughts?

Comment (rosdb) [reply src/core/css-bundler.ts:115]: Let us use the naming used by Lightning CSS, `Draft Syntax`, and update the property too. WDYT? @giotramu

Comment (giotramu) [reply src/core/css-bundler.ts:115]: Sticking to the _Specs_ namespace is better, as this is the [official terminology](https://www.w3.org/Style/CSS/specs.en.html) used by the W3C.

So I suggest `isDraftSpecsIncluded` and `includeDraftSpecs`.

**Comment by rosdb** [comment]
@giotramu, requests have been handled! ✅

**Review by giotramu — APPROVED** [review]

**Pull request merged**